### PR TITLE
delete an unnecessary printing

### PR DESCRIPTION
**Pull request by ceciliazhang12**
delete an unnecessary printing statement
clarify the usage of query in get_products methods

### CODE REVIEW

File: shopcart.py [hunk 1]
```diff
@@ -138,7 +138,7 @@ def get_shopcart(sid):
 ######################################################################
 # RETRIEVE USER'S PRODUCTS LIST IN THE CART
 ######################################################################
-# USAGE: /shopcarts/3/products or /shopcarts/3/products?sku=114672050 for querying for a product
+# USAGE: /shopcarts/3/products or /shopcarts/3/products?name=Settlers of Catan for querying for a product
```
Comment (Pratik-Karnik) [inline shopcart.py:141]: We can even query by sku right? so shouldnt we keep both? @WesPainter @jiweix @ceciliazhang12

Comment (WesPainter) [reply shopcart.py:141]: You can't query by sku as a parameter because its equivalent to /shopcarts/3/products/114672050

Comment (Pratik-Karnik) [reply shopcart.py:141]: Oh I see what you are saying

Comment (ceciliazhang12) [reply shopcart.py:141]: I think query by sku is covered in get_product method, not get_products.

**Review by WesPainter — APPROVED** [review]

**Pull request merged**